server/build.gradle

```diff
@@ -96,9 +96,9 @@ dependencies {
   api "joda-time:joda-time:${versions.joda}"
 
   // percentiles aggregation
-  api 'com.tdunning:t-digest:3.3'
-  // precentil ranks aggregation
-  api 'org.hdrhistogram:HdrHistogram:2.2.1'
+  api "com.tdunning:t-digest:${versions.tdigest}"
+  // percentile ranks aggregation
+  api "org.hdrhistogram:HdrHistogram:${versions.hdrhistogram}"
```

Comment:
This change conflicts with the [PR](hdrhistogram) which bumps `hdrhistogram` to 2.2.2, maybe we can bump the version in this PR and close that older PR.

Reply:
Sure, let me refactor this one